.github/ISSUE_TEMPLATE.md

```diff
@@ -1,89 +1,68 @@
-
+First and foremost, we’d like to thank you for taking the time to contribute to our project. Before submitting your issue, please follow these steps:
+
+1. Familiarize yourself with our contributing guide:
+	* https://github.com/desktop/desktop/blob/master/CONTRIBUTING.md#contributing-to-github-desktop
+2. Check if your issue is in the known-issues and work-arounds doc:
+	* https://github.com/desktop/desktop/blob/master/docs/known-issues.md
+3. Make sure your issue isn’t a duplicate of another issue
+4. If you have made it to this step, go ahead and fill out the template below
 -->
 
-### Description
-
-[Description of the issue]
-
-### Version
-
+## Description
 <!--
-
-What version of GitHub Desktop are you running? This is displayed under the
-`About GitHub Desktop` menu item. If you are running from source, include
-the commit by running `git rev-parse HEAD` from your local repository.
-
+Provide a detailed description of the behavior you're seeing or the behavior you'd like to see **below** this comment.
 -->
 
-**GitHub Desktop version:** [version here]
 
+## Version
 <!--
-
-The operating system you are running on may also help with reproducing the
-issue:
-
- - If you are on macOS, launch `About This Mac` and write down the OS version
-   listed.
- - If you are on Windows, open `Command Prompt` and attach the output of this
-   command: `cmd /c ver`
-
+Place the version of GitHub Desktop you have installed **below** this comment. What version of GitHub Desktop are you running? This is displayed under the 'About GitHub Desktop' menu ite,. If you are running from source, include the commit by running `git rev-parse HEAD` from the local repository.
 -->
-
-**OS version:** [version here]
-
-### Steps to Reproduce
-
-1. [First step]
-1. [Second step]
-1. [and so on]
-
+* GitHub Desktop:
 <!--
+Place the version of your operating system **below** this comment. The operating system you are running on may also help with reproducing the issue. If you are on macOS, launch 'About This Mac' and write down the OS version listed. If you are on Windows, open 'Command Prompt' and attach the output of this command: 'cmd /c ver'
+-->
+* Operating system:
 
-If the issue involves a specific public repository, including the information
-about that repository will make it is easier to recreate the issue.
-
-If you think screenshots or a GIF recording will help demonstrate the issue
-better, feel free to add them here.
 
+## Steps to Reproduce
+<!--
+List the steps to reproduce your issue **below** this comment
+ex,
+1. `step 1`
+2. `step 2`
+3. `and so on…`
 -->
 
-**Expected behavior:** [What you expected to happen]
+### Expected Behavior
+<!-- What you expected to happen -->
 
-**Actual behavior:** [What actually happened]
+### Actual Behavior
+<!-- What actually happens -->
 
-**Reproduces how often:** [What percentage of the time does it reproduce?]
+### How often does this issue occur?
+<!-- `Sometimes`, `Most of the time`, or `Every time` -->
```

Comment:
Part of me just wants to :fire: this section because it doesn't seem to add much value, but I won't hold up the PR for it.

Reply:
👍 People will probably say that they can’t always reproduce it.

Reply:
i'm 👍 on taking it out. if we realize we really needed it, we'll just put it back in, yeah?

Reply:
From my experiences, it's often easy to tell when something is happening every time - especially if the reporter says so when describing the issue in the report.

And most people just default to "it always happens" here anyway.

Reply:
🔥